SQL/updates/13-14.sql

```diff
@@ -0,0 +1,35 @@
+#Updating the SQL from version 13 to version 14. -Kyet
+
+
+# Add new tables used by backend tooling
+
+CREATE TABLE `ip2group`
+(
+  `ip` varchar (18) NOT NULL,
+  `date` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP NOT NULL,
+  `groupstr` varchar (32) NOT NULL DEFAULT '',
+  PRIMARY KEY (`ip`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+ALTER TABLE `ip2group` ADD INDEX(`groupstr`);
+
+
+# Delete tables that already exist but are not used anywhere in either game or backend code
+
+DROP TABLE IF EXISTS `erro_privacy`;
+DROP TABLE IF EXISTS `population`;
```

Comment:
These arent part of `paradise_schema.sql`, and are remains of the original DB schema from 2013, hence the `erro_` prefix, and thus these lines are not relevant to 99% of servers. I would remove them.